### ISSUE TITLE
build fails as of 8ae11b4

### ISSUE THREAD

**Issue by PolarizedIons** [thiagokokada/nix-alien#56]
as of commit 8ae11b4a03034e9d055ae89be35ef2a0a2d93d4a (with the changes from a8f5bc517f966eb5db72b150c5155567186065c8 to the readme) without flakes, fails to build.

```
error: attribute 'inputs' missing

       at /nix/store/f30s5i4qssddab5xxw9yx3hxssqc4b3y-source/default.nix:18:17:

           17|     inherit rev;
           18|     nix-index = self.inputs.nix-index-database.packages.${pkgs.system}.nix-index-with-db;
             |                 ^
           19|   };
(use '--show-trace' to show detailed location information)
```

using the commit before that for now: https://github.com/PolarizedIons/nixos-config/commit/4b6f6cbb4650c4640d182545e5c8ae5948f036e2

not sure what other information to provide.

**Comment by thiagokokada** [comment]
I just reviewed the code and I know where the issue is. Not sure how to solve it exactly yet, but I think I can fix by using `flake-compat`.

Also, I probably need to setup a CI for the non-flake path to avoid those issues in the future.

**Comment by thiagokokada** [comment]
Fixed by #57.
